src/targetsomstandalone.py

```diff
@@ -17,7 +17,7 @@ def entry_point(argv):
     except Exception, e:
         os.write(2, "ERROR: %s thrown during execution.\n" % e)
         return 1
-    return 0
+    return 1
```

Comment:
From what I read in the source code, by default the _avoid_exit attribute in Universe object is set to False, this causes the exit method in Universe class to raise an Exit exception. in Universe.main, the universe object is created with default parameters. That’s why I’m returning 1 if it didn’t raise any exception.
In any case, do you want me to change it to 0?